daemon/daemon_unix.go

```diff
@@ -550,7 +550,7 @@ func verifyPlatformContainerSettings(daemon *Daemon, hostConfig *containertypes.
 	// check for various conflicting options with user namespaces
 	if daemon.configStore.RemappedRoot != "" && hostConfig.UsernsMode.IsPrivate() {
 		if hostConfig.Privileged {
-			return warnings, fmt.Errorf("Privileged mode is incompatible with user namespaces")
+			return warnings, fmt.Errorf("Privileged mode is incompatible with user namespaces.  To temporarily disable user namespaces, use '--userns=host'.")
```

Comment:
I'm open to any suggestions.  This is my first foray into the docker repo.  The issue was raised by a couple of our quality folks and we ground out sometime before we figure out it was a 'feature' rather than a bug, so we're trying to avoid more grinding for folks.